### PR TITLE
Fix phpstan

### PR DESCRIPTION
**Pull request by VincentLanglet**

### CODE REVIEW

**Comment by VincentLanglet** [comment]
Because the option `services` is defined this way
```
->addOption('services', 's', InputOption::VALUE_OPTIONAL, 'The services YAML file', 'services.yaml')
```
psalm consider it's never null.

In the other way, phpstan consider it can be null|string. Who is right ?
How can I solve it ?

I think you solve a similar issue with command of MediaBundle @jordisala1991

**Comment by VincentLanglet** [comment]
Since we're calling `setOption('services')` in the `interact` method, there is no need to put a default value.

**Comment by jordisala1991** [comment]
> Because the option `services` is defined this way
> 
> ```
> 
> ->addOption('services', 's', InputOption::VALUE_OPTIONAL, 'The services YAML file', 'services.yaml')
> 
> ```
> 
> psalm consider it's never null.
> 
> 
> 
> In the other way, phpstan consider it can be null|string. Who is right ?
> 
> How can I solve it ?
> 
> 
> 
> I think you solve a similar issue with command of MediaBundle @jordisala1991 

If you dont want the value to allow null you should make it required, that wont make mandatory to define the parameter but to give it a value when you do it. So you will end with type string. Psalm is wrong here, with original definition you can do --services without value and you will get null here

**Comment by VincentLanglet** [comment]
Since the code in the interact method is
```
$input->setOption('services', false);
        if ($io->confirm('Do you want to update the services YAML configuration file?', true)) {
            $path = sprintf('%s/config/', $this->projectDirectory);
            $servicesFile = $io->ask(
                'The services YAML configuration file',
                is_file($path.'admin.yaml') ? 'admin.yaml' : 'services.yaml',
                [Validators::class, 'validateServicesFile']
            );
            $id = $io->ask(
                'The admin service ID',
                $this->getAdminServiceId($this->adminClassBasename),
                [Validators::class, 'validateServiceId']
            );
            $input->setOption('services', $servicesFile);
            $input->setOption('id', $id);
        }
```

I'm not sure the (default) value of the option `services` is really usefull.
First we set it to `false`, then we're asking the value and setting it.

So am I wrong or the default value was useless ?

File: src/Maker/AdminMaker.php [hunk 1]
```diff
@@ -168,7 +168,7 @@ public function interact(InputInterface $input, ConsoleStyle $io, Command $comma
             );
             $input->setOption('controller', $this->controllerClassBasename);
         }
-        $input->setOption('services', false);
+        $input->setOption('services', null);
```
Comment (jordisala1991) [inline src/Maker/AdminMaker.php:171]: Does this line makes sense ? What happens if you try to pass a --service file?

Comment (VincentLanglet) [reply src/Maker/AdminMaker.php]: The line was useless indeed, I removed it.

**Review by jordisala1991 — APPROVED** [review]

**Pull request merged**